LocalSettings.php

```diff
@@ -3936,6 +3936,9 @@
 			'editextendedconfirmedprotected',
 			'edittemplateprotected',
 		],
+		'+projectsekaiwiki' => [
```

Comment:
They did not request a restriction level.

Reply:
> They did not request a restriction level.

I thought this was required?

Reply:
removed